src/components/CityComponent.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import { StyleSheet, View, Text, ImageBackground } from 'react-native';
+import { utcToZonedTime, format } from 'date-fns-tz';
+import { ru } from 'date-fns/locale';
+import colors from '@styles/colors';
+import { Signs } from '@utils/constants';
+
+export interface ICityComponent {
+    location: string;
+    timeZone: string;
+}
+const getLocalTime = (timeZone: string) => {
+    const currentDate = new Date();
+    return format(utcToZonedTime(currentDate, timeZone), 'HH:mm', { locale: ru });
+};
+
+const initialCityData = {
+    weatherType: 'Облачно',
+    temperature: 12,
+    maxTemperature: 12,
+    minTemperature: -3,
+    image: require('@assets/icons/backgroundRain.png'),
+};
+
+export const CityComponent: React.FC<ICityComponent> = props => {
+    const [cityData, setCityData] = useState(initialCityData);
+    return (
+        <ImageBackground source={cityData.image} style={styles.wrapper} imageStyle={styles.backgroundImage}>
+            <View style={styles.mainContent}>
+                <View>
+                    <Text style={styles.city_title}>{props.location}</Text>
+                    <Text style={styles.time}>{getLocalTime(props.timeZone)}</Text>
+                </View>
+                <Text style={styles.temperature}>{cityData.temperature + Signs.CELSIUS}</Text>
+            </View>
+            <View style={styles.bottomContent}>
+                <Text style={styles.weatherType}>{cityData.weatherType}</Text>
+                <View style={styles.minMaxContainer}>
+                    <Text style={styles.minMaxTemperature}>{'Макс: ' + cityData.maxTemperature + Signs.CELSIUS}</Text>
+                    <Text style={styles.minMaxTemperature}>{'Мин: ' + cityData.minTemperature + Signs.CELSIUS}</Text>
+                </View>
+            </View>
+        </ImageBackground>
+    );
+};
+
+const styles = StyleSheet.create({
+    wrapper: {
+        width: '87%',
+        height: '13%',
```

Comment:
It is better to set height to absolute value, width to 100%, we will be able to set paddings as we need when using this component